src/tools/pub/cmd/pub-global.md

```diff
@@ -110,8 +108,42 @@ and passes in two arguments.
 $ pub global run foo:bar arg1 arg2
 {% endprettify %}
 
+
 ### Running a script from your PATH
 
+In order to run a script directly from your path, it must be specified
+under the `executables` tag in its pubspec file.
```

Comment:
Mentioning the details of the pubspec here may still be a little technical. If I'm a package consumer, it doesn't really matter to me *how* a package provides an executable, it just matters which executables it provides.